src/gfx/error.rs

```diff
@@ -159,32 +174,34 @@ impl<S> NeededBy for StdResult<S, glium::DrawError> {
 
     fn needed_by(self, by: &str) -> Result<Self::Success> {
         use glium::DrawError::*;
-        self.map_err(|e| match e {
-            e@ViewportTooLarge |
-            e@UnsupportedVerticesPerPatch |
-            e@TessellationNotSupported |
-            e@SamplersNotSupported |
-            e@TransformFeedbackNotSupported |
-            e@SmoothingNotSupported |
-            e@ProvokingVertexNotSupported |
-            e@RasterizerDiscardNotSupported |
-            e@DepthClampNotSupported |
-            e@BlendingParameterNotSupported => Error::UnsupportedFeature {
-                                                   feature: format!("{:?}", e),
-                                                   needed_by: by.to_owned(),
-                                               },
-            e@NoDepthBuffer |
-            e@AttributeTypeMismatch |
-            e@AttributeMissing |
-            e@InvalidDepthRange |
-            e@UniformTypeMismatch { .. } |
-            e@UniformBufferToValue { .. } |
-            e@UniformValueToBlock { .. } |
-            e@UniformBlockLayoutMismatch { .. } |
-            e@TessellationWithoutPatches |
-            e@InstancesCountMismatch |
-            e@VerticesSourcesLengthMismatch |
-            e@WrongQueryOperation => panic!("Invalid draw call: {:?}", e),
+        self.map_err(|e| {
+            match e {
+                e@ViewportTooLarge |
+                e@UnsupportedVerticesPerPatch |
+                e@TessellationNotSupported |
+                e@SamplersNotSupported |
+                e@TransformFeedbackNotSupported |
+                e@SmoothingNotSupported |
+                e@ProvokingVertexNotSupported |
+                e@RasterizerDiscardNotSupported |
+                e@DepthClampNotSupported |
+                e@BlendingParameterNotSupported => Error::UnsupportedFeature {
+                    feature: format!("{:?}", e),
+                    needed_by: by.to_owned(),
+                },
```

Comment:
It'd be nice to have a space here to separate the two groups, similar to the way it's done above.